fuel/app/tests/helper/common.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @group Helper
+ */
+class Test_Helper_Common extends TestCase
+{
+	/**
+	 * @before
+	 */
+	protected function truncate_data()
+	{
+		DBUtil::truncate_table('messages');
+	}
+
+	public function test_format_time()
+	{
+		$expect = "10/05/2015 12:10";
+		$date   = new DateTime($expect);
+		$date   = $date->getTimestamp();
+
+		$this->assertEquals(Helper_Common::format_time($date), $expect);
+	}
+
+	/**
+	 * @dataProvider data_insert_data
+	 */
+	public function test_insert_data($input,$expect)
+	{
+		Helper_Common::install_data($input['data'],$input['table']);
+		$result = Model_Message::get_all();
+
+		$this->assertEquals($expect, count($result));
+	}
+
+	public function data_insert_data()
```

Comment:
@kondo-takuto-mulodo 
ah It's used for dataset branch to insert data from csv, json or xml to database. Sorry, I forgot move it to dataset branch.